project/build.scala

```diff
@@ -333,21 +325,25 @@ object ScalatraBuild extends Build {
                                                                                     ).map(_        % specs2Version)
     lazy val testJettyServlet         =  "org.eclipse.jetty"       %  "test-jetty-servlet"         % jettyVersion
     lazy val testng                   =  "org.testng"              %  "testng"                     % "6.9.9" exclude("junit", "junit")
-    lazy val metricsScala             =  "nl.grons"                %% "metrics-scala"              % "3.5.2"
+    def metricsScala(version: String) = if(version.startsWith("2.11")){
+      "nl.grons" %% "metrics-scala" % "3.5.5"
+    } else {
+      "nl.grons" %% "metrics-scala" % "3.5.6-snapshot"
+    }
     lazy val metricsServlets          =  "io.dropwizard.metrics"   %  "metrics-servlets"           % "3.1.2"
     lazy val metricsServlet           =  "io.dropwizard.metrics"   %  "metrics-servlet"            % "3.1.2"
     lazy val googleGuava              =  "com.google.guava"        % "guava"                       % "19.0"
     lazy val googleFindBugs           = "com.google.code.findbugs" % "jsr305"                      % "3.0.1"
 
-    private val akkaVersion             = "2.3.15" // 2.4 dropped Java 7
-    private val grizzledSlf4jVersion    = "1.0.4"
+    private val akkaVersion             = "2.4.12"
+    private val grizzledSlf4jVersion    = "1.3.0"
     private val atmosphereCompatVersion = "2.0.1"
     private val httpcomponentsVersion   = "4.5.2"
     private val jettyVersion            = "9.2.17.v20160517"
-    private val json4sVersion           = "3.3.0"
-    private val scalateVersion          = "1.7.1"
-    private val scalatestVersion        = "2.2.5"
-    private val specs2Version           = "3.7.3"
+    private val json4sVersion           = "3.5.0.RC1"
```

Comment:
3.5.0 is latest version

https://oss.sonatype.org/content/repositories/releases/org/json4s/json4s-core_2.12/